### PR TITLE
Introductory workflow + workflow section

### PR DESCRIPTION
**Pull request by AxelDgn**
Currently, OMA chapters focus on individual methods/analyses. In order to simplify and guide its use, here's an introductory workflow for the most basic methods. From importing data to understanding its structure and handling (transformation, subsetting, aggregation,...).

It's not finished yet we still have to add some content.

### CODE REVIEW

File: inst/pages/Introductory-workflow/Introductory.qmd [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+```{r}
+#| echo: false 
+#| output: false
+library(DT)
+```
+
+You can either choose to use your own data or one of the built-in datasets provided by mia which you may find in the [**OMA book**](https://microbiome.github.io/OMA/docs/devel/pages/04_containers.html#sec-example-data) :
+
+In this tutorial, we'll be using the [Tengeler2020](https://microbiome.github.io/mia/reference/Tengeler2020.html) dataset. This dataset was made by A.C. Tengeler to try to demonstrate the impact of altered microbiomes on the brain structure. Here's how we can load the data into our R environment :
```
Comment (antagomir) [inline inst/pages/Introductory-workflow/Introductory.qmd]: remove spaces before ":" everywhere

File: inst/pages/Introductory-workflow/Introductory.qmd [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+
+```{r}
+data("Tengeler2020", package="mia")
+tse <- Tengeler2020
+```
+
+There are of course other ways to import your data using the [**mia package**](https://microbiome.github.io/mia/). These include: using either [**your own data**](https://microbiome.github.io/OMA/docs/devel/pages/04_containers.html#sec-import-from-file) or by converting an existing object to a *TreeSummarizedExperiment* object as stated in [**this section of the OMA book**](https://microbiome.github.io/OMA/docs/devel/pages/04_containers.html#conversions-between-data-formats-in-r).
```
Comment (antagomir) [inline inst/pages/Introductory-workflow/Introductory.qmd]: Dont use direct links to cite OMA sections. Use automated referencing instead. See some other chapters for examples. This way the sections will be cited by automatically updated section numbers.

File: inst/pages/Introductory-workflow/Introductory.qmd [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+
+<hr>
+
+# Storing microbial data
```
Comment (antagomir) [inline inst/pages/Introductory-workflow/Introductory.qmd]: microbial -> microbiome

File: inst/pages/Introductory-workflow/Introductory.qmd [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+
+*TreeSummarizedExperiment* or TSE objects is the type of object used within the [**mia package**](https://microbiome.github.io/mia/) to store your data. It's a versatile and multi-purpose data type that allows for an efficient way to store and access data.
```
Comment (antagomir) [inline inst/pages/Introductory-workflow/Introductory.qmd]: objects -> object

File: inst/pages/Introductory-workflow/Introductory.qmd [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+
+```{r}
+assay(tse)[1:5,1:10] 
+```
+
+## colData
+
+Another important aspect of microbiome analysis is sample data.
+
+```{r}
```
Comment (antagomir) [inline inst/pages/Introductory-workflow/Introductory.qmd]: add some comments that explain what the code does

File: inst/pages/Introductory-workflow/Introductory.qmd (hunk 3 above)
Comment (antagomir) [inline inst/pages/Introductory-workflow/Introductory.qmd]: This section could be shortened (remove subsections on assays, colData, rowData), see if something could be moved to OMA chapter that deals with these and then here just provide very brief examples and refer the reader there.

File: inst/pages/Introductory-workflow/Introductory.qmd [hunk 6]
```diff
@@ -0,0 +1,293 @@
...
+#| echo: false 
+#| output: false
+options(warn = 0)
+```
+
+Here `rowData(tse)` returns a DataFrame with 151 rows and 7 columns. Each row represents an organism and each column a taxonomic level.
+
+In the example, ***Kingdom*** indicates whether the feature belongs to the kingdom of bacteria or archaea, ***Phylum*** indicates the phylum to which the individual belongs and so on down to the specific species.
+
+To illustrate the structure of a *TreeSummarizedExperiment*, here's [**an article**](https://f1000research.com/articles/9-1246) that uses this type of object. Also, please take a look at figure 1 below.
```
Comment (antagomir) [inline inst/pages/Introductory-workflow/Introductory.qmd]: use bibtex citation instead, see other chapters for examples

File: inst/pages/Introductory-workflow/Introductory.qmd [hunk 7]
```diff
@@ -0,0 +1,293 @@
...
+
+In some cases, you may only need to use part of your original *TreeSummarizedExperiment*.
+
+Using the Tengeler2020 dataset, we can focus on a certain cohort for example. Doing this is pretty straightforward:
+
+```{r}
+tse_subset_by_sample <- tse[ , tse$cohort =="Cohort_1"]
+```
+
+This will create a *TreeSummarizedExperiment* Object only containing samples of the first cohort. To visualize the resulting object as a dataframe we can use the `meltAssay` method and specify what columns will be put into the dataframe:
```
Comment (antagomir) [inline inst/pages/Introductory-workflow/Introductory.qmd]: melting example could better go to the chapter in OMA that mentions meltAssay

File: inst/pages/Introductory-workflow/Introductory.qmd [hunk 8]
```diff
@@ -0,0 +1,293 @@
...
+
+```
+
+It's very important to make all these comparisons in order to quantify diversity and compare samples in our data using different measures. - You can find other types of comparisons directly in the book [**here**](https://microbiome.github.io/OMA/docs/devel/pages/14_alpha_diversity.html).
+
+<hr>
+
+## Community similarity
+
+Community similarity refers to the way microorganisms resemble each other in terms of their composition and abundance of different microbial taxa. This can help us understand to what degree different samples resemble each other and finding key information. In microbiome analysis however, it's more common to measure the dissimilarity/Beta diversity between two samples A and B using the bray-Curtis measure which is defined as follows:
```
Comment (antagomir) [inline inst/pages/Introductory-workflow/Introductory.qmd]: bray -> Bray

File: inst/pages/Introductory-workflow/Introductory.qmd [hunk 9]
```diff
@@ -0,0 +1,293 @@
...
+tse <- transformAssay(tse,
+                      assay.type = "counts",
+                      method = "relabundance")
+```
+
+This will take the original counts assay and apply the Bray-Curtis formula to it. The result is a matrix with the sample identifiers as rows and the dissimilarity to other samples as columns. It can be accessed in the assays of the tse:
+
+```{r}
+assay(tse, "relabundance")[5:10,1:10]
+```
```
Comment (antagomir) [inline inst/pages/Introductory-workflow/Introductory.qmd]: repetition?

File: inst/pages/Introductory-workflow/Introductory.qmd (hunk 4 above)
Comment (TuomasBorman) [reply inst/pages/Introductory-workflow/Introductory.qmd]: Also TSE --> TreeSE

File: inst/assets/book.scss [hunk 10]
```diff
@@ -168,6 +168,200 @@ li.dropdown-header > span.menu-text {
...
+		max-width: 70%;
+		max-height: auto;
+		text-align: center; 
+		margin: 0 auto; 
+	}
+
+	#quarto-content{
+	  margin: 0;
+	  padding: 0;
+	  width: 100%;
```
Comment (TuomasBorman) [inline inst/assets/book.scss]: What these does?

Comment (TuomasBorman) [reply inst/assets/book.scss]: Why the default is not working?

File: inst/pages/Introductory-workflow/Introductory.qmd
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow/Introductory.qmd:1]: Is there are reason why this is in its own directory?

File: inst/pages/Introductory-workflow/Introductory.qmd (hunk 7 above)
Comment (TuomasBorman) [reply inst/pages/Introductory-workflow/Introductory.qmd]: Also meltAssay is now meltSE

File: inst/pages/Introductory-workflow/images/TSE-structure.svg
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow/images/TSE-structure.svg:1]: Cool! It is good that we have different ways to visualize the object.

1. The package names should be in camel case
2. There is assay(TSE) even though other use variable name "tse"
3. Info could be patient info which might tell more

File: inst/pages/Introductory-workflow/Introductory.qmd [hunk 11]
```diff
@@ -0,0 +1,293 @@
...
+
+# Reonvert to an interactive plot with ggplotly
+ggplotly(p)
+```
+
+There we have it! Each axis shows the amount of variance or in our case dissimilarity retained by each principal coordinate. You can additionally add more options to color by a certain characteristic for example. More on this can be found in [**this section**](https://microbiome.github.io/OMA/docs/devel/pages/20_beta_diversity.html#comparing-communities-by-beta-diversity-analysis) of the OMA book.
+
+<hr>
+
+# Datavisualization
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow/Introductory.qmd]: Data visualization

File: inst/assets/book.scss (hunk 10 above)
Comment (AxelDgn) [reply inst/assets/book.scss]: > What these does?

It's to give a different style to the workflow

Comment (TuomasBorman) [reply inst/assets/book.scss]: OK good, does it affect other sections?

File: inst/pages/Introductory-workflow/Introductory.qmd [hunk 12]
```diff
@@ -0,0 +1,293 @@
...
+
+In some cases, you may need to change your data in order to get your wanted results. In this section we'll go over how to agglomerate data, subset your data and more. A *TreeSummarizedExperiment* allows for some neat data manipulation using [**the dplyr package**](https://dplyr.tidyverse.org/).
+
+## Subsetting
+
+In some cases, you may only need to use part of your original *TreeSummarizedExperiment*.
+
+Using the Tengeler2020 dataset, we can focus on a certain cohort for example. Doing this is pretty straightforward:
+
+```{r}
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow/Introductory.qmd]: Name all chunks

File: inst/pages/Introductory-workflow/Introductory.qmd [hunk 13]
```diff
@@ -0,0 +1,293 @@
...
+assay(tse, "relabundance")[5:10,1:10]
+```
+
+In our case, the assay contains 151 rows and 27 columns. Having so many columns and thus dimensions can be troublesome for visualizing dissimilarity.
+
+In order to visualize the dissimilarity between the different samples, we can conduct a Principal Coordinate analysis on the newly created assay. This essentially projects the Bray-curtis dimensions onto a lower space whilst maintaining as much of the variation as possible, the projected values are called principal coordinates. You can read more [**here**](https://en.wikipedia.org/wiki/Multidimensional_scaling#Types).
+
+MIA doesn't provide a direct way to reduce dimensionality but we can use Bioconductor's scater package and the vegan package, the latter can be found [**here**](https://cran.r-project.org/web/packages/vegan/vegan.pdf) to turn the dissimilarity into actual distances that can be visualized:
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow/Introductory.qmd]: mia, not MIA. mia has actually some dimension reduction techniques available such as dbRDA

File: inst/pages/Introductory-workflow/Introductory.qmd [hunk 14]
```diff
@@ -0,0 +1,293 @@
...
+
+$$
+BC_{ij} = \frac{\sum_{k} |A_{k} - B_{k}|}{\sum_{k} (A_{k} + B_{k})}
+$$ Luckily for us, the [**mia package**](https://microbiome.github.io/mia/) provides an easy way to calculate the Bray-Curtis dissimilarity and add it to our TreeSE using the `transformAssay` method:
+
+```{r}
+tse <- transformAssay(tse,
+                      assay.type = "counts",
+                      method = "relabundance")
+```
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow/Introductory.qmd]: This does not calculate BC, it just calculated relative abundance

File: inst/pages/Introductory-workflow/Introductory.qmd [hunk 15]
```diff
@@ -0,0 +1,293 @@
...
+# Load package to plot reducedDim
+library(scater)
+
+# Run PCoA on relabundance assay with Bray-Curtis distances
+tse <- runMDS(tse,
+              FUN = vegan::vegdist,
+              method = "bray",
+              assay.type = "relabundance",
+              name = "MDS_bray")
+```
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow/Introductory.qmd]: This applies BC to relative abundances

File: inst/pages/Introductory-workflow/Introductory.qmd [hunk 16]
```diff
@@ -0,0 +1,293 @@
...
+# Makes the plots interactive
+library(plotly)
+```
+
+```{r}
+# Create ggplot object
+p <- plotReducedDim(tse, "MDS_bray") +
+  geom_point(color = "#659b91")
+
+# Convert to an interactive plot with ggplotly
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow/Introductory.qmd]: You could show 3 dimensional plot since you are using plotly

Comment (TuomasBorman) [reply inst/pages/Introductory-workflow/Introductory.qmd]: Also might be good example to who how color those points with variables from colData

**Comment by antagomir** [comment]
Status of this one?

**Comment by AxelDgn** [comment]
> Status of this one?

I'm working on it but I was focused on my internship report, so it's progressing slowly, it'll be done for the next week

File: inst/pages/Introductory-workflow/Introductory.qmd (hunk 16 above)
Comment (AxelDgn) [reply inst/pages/Introductory-workflow/Introductory.qmd]: We discussed this with Leo and Noah has moved the 3d version of this plot to OMA so that we can keep an easier 2d version in the workflow.

Comment (AxelDgn) [reply inst/pages/Introductory-workflow/Introductory.qmd]: Should I still color the points ?

File: inst/pages/Introductory-workflow/Introductory.qmd (hunk 3 above)
Comment (AxelDgn) [reply inst/pages/Introductory-workflow/Introductory.qmd]: I shortened this section. However wouldn't it be beneficial to keep some longer explanation in the French and Dutch version of the workflow ?

File: inst/pages/Introductory-workflow.qmd [hunk 17]
```diff
@@ -0,0 +1,353 @@
...
+#| output: false
+library(mia)
+```
+
+```{r}
+#| label: loadDT
+#| echo: false 
+#| output: false
+library(DT)
+```
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow.qmd]: Better to have one chunk that loads all the required packages
Similar to this
```

```{r setup}
# List of packages that we need
packages <- c(
    "ggplot2", "knitr", "mia", "dplyr", "miaViz", "MOFA2",  "patchwork", "reticulate",
    "MGnifyR", "reshape2", "GGally", "IntegratedLearner", "SuperLearner",
    "bartMachine", "mcmcplots", "tidyverse", "UpSetR", "shadowtext", "ComplexHeatmap",
    "stringr", "cowplot", "caret", "scater", "ggsignif"
    )

# Get packages that are already installed installed
packages_already_installed <- packages[ packages %in% installed.packages() ]

# Get packages that need to be installed
packages_need_to_install <- setdiff( packages, packages_already_installed )

# Loads BiocManager into the session. Install it if it not already installed.
if( !require("BiocManager") ){
    install.packages("BiocManager")
    library("BiocManager")
}

# If there are packages that need to be installed, installs them with BiocManager
# Updates old packages.
if( length(packages_need_to_install) > 0 ) {
   install(packages_need_to_install, ask = FALSE)
}

# Load all packages into session. Stop if there are packages that were not
# successfully loaded
pkgs_not_loaded <- !sapply(packages, require, character.only = TRUE)
pkgs_not_loaded <- names(pkgs_not_loaded)[ pkgs_not_loaded ]
if( length(pkgs_not_loaded) > 0 ){
    stop("Error in loading the following packages into the session: '", paste0(pkgs_not_loaded, collapse = "', '"), "'")
}
```
```

File: inst/pages/Introductory-workflow.qmd [hunk 18]
```diff
@@ -0,0 +1,353 @@
...
+
+[**rowData**](https://microbiome.github.io/OMA/docs/devel/pages/04_containers.html#rowdata) contains data on sample characteristics, particularly taxonomic information.
+
+```{r}
+#| label: setWarning
+#| echo: false 
+#| output: false
+options(warn = -1)
+```
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow.qmd]: Instead, use warning=FALSE in a chunk where you want to silence warnings https://quarto.org/docs/computations/execution-options.html

File: inst/pages/Introductory-workflow.qmd [hunk 19]
```diff
@@ -0,0 +1,353 @@
...
+tse_alpha <- Tengeler2020
+
+# Estimate (observed) richness
+tse_alpha <- mia::estimateRichness(tse_alpha, 
+                             assay.type = "counts", 
+                             index = "observed", 
+                             name="observed")
+
+# Check some of the first values in colData
+head(tse_alpha$observed)
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow.qmd]: Use the `tse` variable that you already used above

File: inst/pages/Introductory-workflow.qmd [hunk 20]
```diff
@@ -0,0 +1,353 @@
...
+#| label: plotColdata
+plotColData(tse_alpha, 
+            "observed", 
+            "cohort", 
+            colour_by = "patient_status") +
+  theme(axis.text.x = element_text(angle=45,hjust=1)) + 
+  labs(y=expression(Richness[Observed]))
+```
+
+To go even further, we can also compare the estimated Shannon index to the observed richness . Shannon quantifies the diversity in terms of both the number of different species (richness) and the evenness of their distribution (abundance) and is calculated as such:
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow.qmd]: "observed richness ." extra space

File: inst/pages/Introductory-workflow.qmd [hunk 21]
```diff
@@ -0,0 +1,353 @@
...
+H' = -\sum_{i=1}^{R} p_i \ln(p_i)
+$$ p~i~ being the proportion of a certain microorganism.
+
+<hr/>
+
+First, we can easily calculate this measure and add them to our TreeSE.
+
+```{r}
+#| label: calculateDiversity
+tse_alpha <- mia::estimateDiversity(tse_alpha,
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow.qmd]: Sometimes you use package::function and sometimes not. Might be better to not use

File: inst/pages/Introductory-workflow.qmd [hunk 22]
```diff
@@ -0,0 +1,353 @@
...
+
+# Run PCoA on relabundance assay with Bray-Curtis distances
+tse <- runMDS(tse,
+              FUN = vegan::vegdist,
+              method = "bray",
+              assay.type = "relabundance",
+              name = "MDS_bray")
+```
+
+
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow.qmd]: Remove extra spaces

File: inst/pages/Introductory-workflow.qmd [hunk 23]
```diff
@@ -0,0 +1,353 @@
...
+# Reonvert to an interactive plot with ggplotly
+ggplotly(p)
+```
+
+There we have it! Each axis shows the amount of variance or in our case dissimilarity retained by each principal coordinate. You can additionally add more options to color by a certain characteristic for example. More on this can be found in @sec-comparing-communities-by-beta-diversity-analysis.
+
+<hr>
+
+# Datavisualization using heatmaps
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow.qmd]: Datavisualization

File: inst/pages/Introductory-workflow.qmd [hunk 24]
```diff
@@ -0,0 +1,353 @@
...
+#| label: load-sechm
+#| output: false
+library(sechm)
+```
+
+Then, let's subset the TreeSE to the most prevalent taxa:
+
+```{r}
+#| label: most-prevalent
+tse.subset <- subsetByPrevalent(tse,prevalence=0.5)[1:5,]
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow.qmd]: Use altExp()

File: inst/pages/Introductory-workflow.qmd [hunk 25]
```diff
@@ -0,0 +1,353 @@
...
+
+<h2>Acknowledgements</h2>
+
+This notebook has been carefully crafted by:
+
+![](images/profile-pics/axel.webp){.profilepic} **Axel Dagnaud**
+
+![](images/profile-pics/noah.png){.profilepic} **Noah de Gunst**
+
+-   **Leo Lahti**
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow.qmd]: All the authors are listed in https://microbiome.github.io/OMA/docs/devel/pages/90_acknowledgments.html so add your names there instead if they are not there already

**Review by TuomasBorman — CHANGES_REQUESTED** [review]

File: inst/pages/Introductory-workflow.qmd [hunk 26]
```diff
@@ -0,0 +1,353 @@
+# Introductory
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow.qmd]: It seems that the chapter starts with "Chapter 2 - Introductory". I guess this should be "## Introductory" (Render the book as check that everything is correct)

File: inst/pages/images/logo.svg
Comment (TuomasBorman) [inline inst/pages/images/logo.svg:1]: Rename the file as "intro_workflow_logo.svg" or something

File: inst/pages/Introductory-workflow.qmd [hunk 27]
```diff
@@ -0,0 +1,353 @@
...
+
+In this tutorial, we'll be using the @Tengeler2020 dataset. This dataset was made by A.C. Tengeler to try to demonstrate the impact of altered microbiomes on the brain structure. Here's how we can load the data into our R environment:
+
+```{r}
+#| label: loadDataset
+data("Tengeler2020", package="mia")
+tse <- Tengeler2020
+```
+
+There are of course other ways to import your data using the [**mia package**](https://microbiome.github.io/mia/). These include: using either **your own data** @sec-import-from-file or by converting an existing object to a *TreeSummarizedExperiment* object as stated in @sec-conversions-between-data-formats-in-r.
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow.qmd]: All links in here seems not to be working

**Review by TuomasBorman — CHANGES_REQUESTED** [review]
Couple of points. I had now time to render this. It looks nice and is probably very useful for many users. Thanks!

File: inst/pages/images/profile-pics/Axel.webp
Comment (TuomasBorman) [inline inst/pages/images/profile-pics/Axel.webp:1]: Is this file still needed

File: inst/assets/book.scss [hunk 28]
```diff
@@ -177,11 +177,12 @@ li.dropdown-header > span.menu-text {
...
-	  /* color: #070707; */
 	  color: #b5b5b5;
 	}
 	.sidebar.sidebar-navigation:not(.rollup) {
-	  /* border-right: 1px solid #434343 !important; */
+	  color: #b5b5b5;
+	}
+	.sidebar.sidebar-navigation:not(.rollup) {
 	  border-right: 1px solid #222 !important;
 	}
```
Comment (TuomasBorman) [inline inst/assets/book.scss]: This seems to be duplicated?

File: inst/assets/book.scss [hunk 29]
```diff
@@ -227,9 +228,148 @@ li.dropdown-header > span.menu-text {
...
+	  color: #070707;
+	  font-weight: 700;
+	  border-image: linear-gradient(to right, #007562, #007562) 1;
+	  border-bottom-style: solid;
+	  border-bottom-width: 4px;
+	  margin-bottom: 10%;
+	}
+	
+	h2 {
+	  margin-top: 3rem;
```
Comment (TuomasBorman) [inline inst/assets/book.scss]: Can you explain what this new format does? Why the default is not working here? I cannot see clear difference between default formatting and this.

Also, if this clearly improves the formatting, this could be implemented in whole book

**Review by TuomasBorman — COMMENTED** [review]
Approvement for the workflow. It's very nice.

I still want comments for the formatting. It should be kept in mind that there will be more workflows in the future. Do they work with this new formatting?

As developing the methods and the content of book take most of our time, we want to rely that the structure and format of book works. That is why we use BiocBook. This format modification is more than welcome, and the already implemented changes in format have already improved OMA.

I want just to make sure that we can maintain this formatting without sudden failures that we cannot solve without extensive amount of work

File: inst/assets/book.scss (hunk 29 above)
Comment (AxelDgn) [reply inst/assets/book.scss]: We wanted to create a style different from OMA to clearly distinguish between the two. There are indeed unnecessary elements, and I will remove them. What this formatting does is quite similar to OMA's, but with different colors and font formatting. The default formatting does not work here because all the page content is contained within a <div id="workflow"> tag, which applies the workflow formatting as a priority to the page.

File: inst/pages/Introductory-workflow/Introductory.qmd (hunk 16 above)
Comment (TuomasBorman) [reply inst/pages/Introductory-workflow/Introductory.qmd]: You could colour the based on sample metadata.

Does ggplotly give any additional value?

File: inst/assets/book.scss (hunk 29 above)
Comment (TuomasBorman) [reply inst/assets/book.scss]: Can you also comment the lines so that it is easier to maintain?

Comment (TuomasBorman) [reply inst/assets/book.scss]: One more thing: Is it possible to specify that this formatting is applied only in this "introductory workflow". In the future, there might be other workflows, where we do not want to apply this formatting?

Have you rendered book (you can temporarily remove chapters from book.yml)? For me it does not look difference between "normal" chapters and workflows. (Looks good though)

File: inst/pages/Introductory-workflow/Introductory.qmd (hunk 16 above)
Comment (AxelDgn) [reply inst/pages/Introductory-workflow/Introductory.qmd]: It adds interactivity to the plot so people can actually look at the individual values.

**Review by antagomir — APPROVED** [review]

File: inst/pages/Introductory-workflow/Introductory.qmd (hunk 3 above)
Comment (antagomir) [reply inst/pages/Introductory-workflow/Introductory.qmd]: Ok

**Comment by antagomir** [comment]
@AxelDgn can we finalize this, seems nearly ready?

**Comment by ElySeraidarian** [comment]
I have removed the workflow formatting, fixed issues that came with it (e.g. flags not centered anymore). Also fixed some typo issues. Not have gone through all the text but I could check if there are small typo issues

File: inst/pages/Introductory-workflow.qmd [hunk 30]
```diff
@@ -17,7 +21,7 @@ Hello and welcome to a complete workflow using the latest R/Bioconductor tools f
 
 When using microbiome packages, there are many different ways to import your data. Let's first load the required package:
 
-```{r setup}
+```{r}
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow.qmd:27]: Can you name this chunk

#| label: install

File: inst/pages/Introductory-workflow-french-version.qmd [hunk 31]
```diff
@@ -17,7 +22,7 @@ Bonjour et bienvenue dans un workflow complet utilisant les derniers outils R/Bi
 
 Lors de l'utilisation de packages pour le microbiome, il existe de nombreuses façons différentes d'importer vos données. Commençons par charger les packages requis:
 
-```{r setup}
+```{r}
 #| output: FALSE
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow-french-version.qmd:30]: Can you name this chunk

#| label: install

File: inst/pages/Introductory-workflow-french-version.qmd [hunk 32]
```diff
@@ -0,0 +1,333 @@
+## Introductory (French)
+
+```{r setup, echo=FALSE, results="asis"}
+library(rebook)
+chapterPreamble()
+```
```
Comment (TuomasBorman) [inline inst/pages/Introductory-workflow-french-version.qmd]: This is now in Rmarkdown-style

Can you  put it in quarto-style?

Can you name this chunk

#| label: setup
#| echo: FALSE
#| results: "asis"

might work

**Review by TuomasBorman — APPROVED** [review]
Great! Let's merge this now and improve it later if needed. Thanks all!
